Tasks/Common/ServiceFabricHelpers/Connect-ServiceFabricClusterFromServiceEndpoint.ps1

```diff
@@ -135,10 +135,10 @@ function Connect-ServiceFabricClusterFromServiceEndpoint {
         # Configure cluster connection pre-reqs
         if ($ConnectedServiceEndpoint.Auth.Scheme -ne "None")
         {
-            # Add server cert thumbprint (common to both auth-types)
+            # Add server cert thumbprint(s) (common to both auth-types)
             if ($ConnectedServiceEndpoint.Auth.Parameters.ServerCertThumbprint)
```

Comment:
The service endpoint help text should also be updated to reflect the fact that multiple thumbprints separated by comma is alllowed now. Do you have access to VSO repo in mseng where service endpoint json code is?

Reply:
I have access to the repo, but i don't know where the endpoints are defined. Could you e-mail me a pointer?

Reply:
It would be this section: https://mseng.visualstudio.com/VSOnline/_git/VSO?path=%2FDistributedTask%2FService%2FExtensionPackages%2FAzure%2Fvss-extension.json&version=GBmaster&_a=contents&line=970&lineStyle=plain&lineEnd=970&lineStartColumn=1&lineEndColumn=194

Reply:
Created a PR https://mseng.visualstudio.com/VSOnline/_git/VSO/pullrequest/300366